test/data/convertors/hexadecimalToBinaryString.json

```diff
@@ -1,4 +1,4 @@
 {
     "from": "0438082601f8b38ae010a621a48f4b4cd021c4e6e69219e3c2d8abab482039e9",
-    "to": "0000010000111000000010000010011000000001111110001011001110001010111000000001000010100110001000011010010010001111010010110100110011010000001000011100010011100110111001101001001000011001111000111100001011011000101010111010101101001000001000000011100111101001"
+    "to": "0000001011000001000000010100011000001000111100011101110000010101011100001000000001010110010010000101001000011111001011010010001110110000010010000011001001110110011101101001010010001001011111000011010010110001010111010101110100100001010000001100100101111001"
```

Comment:
Unless I'm missing something, this binary string is incorrect. For example, the first 8 bits in the string should be `0010_0000`, i.e., the presentation of `0x04` starting from LSB to MSB. Another troubling sign is that roundtrip doesn't work:

```javascript
const { hexadecimalToBinaryString: h2b } = require('exonum')
h2b('201c1064801fcd510708658425f1d2320b842367674998c7431bd5d512049c97')
// outputs '01000000....' rather than '00000100...'
```